### PR TITLE
feat(kit)!: `Prompt` rename to `Confirm` and add appearance prop

### LINKED ISSUE #6986: 🚀 - Defining the style of the yes button in prompt
### Description

We're using [prompt](https://taiga-ui.dev/components/prompt) to show confirmation dialogs.

However, sometimes the action is destructive, and we feel like the [secondary destructive or accent](https://taiga-ui.dev/components/button#appearance) styles would be more appropriate.

It'd be great to be able to configure this.  

Looking at the source for the [prompt.template.html](https://github.com/taiga-family/taiga-ui/blob/main/projects/kit/components/prompt/prompt.template.html) it doesn't appear that this appearance is configurable.

### PR DESCRIPTION
**Pull request by waterplea**
Closes #6986

### CODE REVIEW

**Review by splincode — APPROVED** [review]

**Review by nsbarsukov — APPROVED** [review]

File: projects/demo-playwright/tests/core/dialogs/dialogs.spec.ts
Comment (nsbarsukov) [inline projects/demo-playwright/tests/core/dialogs/dialogs.spec.ts:1]: https://github.com/taiga-family/taiga-ui/actions/runs/8374329098/job/22929527139?pr=7035
```shell
·F
  1) [chromium] › tests/core/dialogs/dialogs.spec.ts:11:13 › Dialogs › Prompt - 320x480 ────────────
    TimeoutError: locator.evaluate: Timeout 10000ms exceeded.
    Call log:
      - waiting for locator('[tuiDocHeader]')
       at utils/hide-element.ts:10
       8 |
       9 | export async function tuiRemoveElement(element: Locator): Promise<void> {
    > 10 |     return element.evaluate(el => el.remove());
         |                    ^
      11 | }
      12 |
        at tuiRemoveElement (/home/runner/work/taiga-ui/taiga-ui/projects/demo-playwright/utils/hide-element.ts:10:20)
        at tuiGoto (/home/runner/work/taiga-ui/taiga-ui/projects/demo-playwright/utils/goto.ts:63:31)
        at /home/runner/work/taiga-ui/taiga-ui/projects/demo-playwright/tests/core/dialogs/dialogs.spec.ts:13:13
    Retry #1 ───────────────────────────────────────────────────────────────────────────────────────
    TimeoutError: locator.evaluate: Timeout 10000ms exceeded.
    Call log:
      - waiting for locator('[tuiDocHeader]')
       at utils/hide-element.ts:10
       8 |
       9 | export async function tuiRemoveElement(element: Locator): Promise<void> {
    > 10 |     return element.evaluate(el => el.remove());
         |                    ^
      11 | }
```

Comment (nsbarsukov) [reply projects/demo-playwright/tests/core/dialogs/dialogs.spec.ts:1]: https://github.com/taiga-family/taiga-ui/blob/79366366c330f54895f6a1d86da06b8e9712562d/projects/demo-playwright/tests/core/dialogs/dialogs.spec.ts#L15

`prompt` => `confirm`

Comment (nsbarsukov) [reply projects/demo-playwright/tests/core/dialogs/dialogs.spec.ts:1]: Снимок экрана 2024-03-22 в 10 31 59

File: projects/kit/services/dialog-form.service.ts [hunk 1]
```diff
@@ -20,11 +20,11 @@ export class TuiDialogFormService {
     }
 
     public withPrompt(
```
Comment (nsbarsukov) [inline projects/kit/services/dialog-form.service.ts:22]:
```suggestion
    public withConfirm(
```

File: projects/kit/tokens/i18n.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ import type {TuiCountryIsoCode} from '@taiga-ui/i18n';
 import {tuiExtractI18n} from '@taiga-ui/i18n';
 import type {Observable} from 'rxjs';
 
-export const TUI_PROMPT_WORDS = tuiCreateTokenFromFactory(tuiExtractI18n('prompt'));
+export const TUI_CONFIRM_WORDS = tuiCreateTokenFromFactory(tuiExtractI18n('prompt'));
```
Comment (nsbarsukov) [inline projects/kit/tokens/i18n.ts:7]:
```suggestion
export const TUI_CONFIRM_WORDS = tuiCreateTokenFromFactory(tuiExtractI18n('confirm'));
```

File: projects/demo/src/modules/app/app.routes.ts
Comment (nsbarsukov) [inline projects/demo/src/modules/app/app.routes.ts:1]: https://github.com/taiga-family/taiga-ui/tree/main/projects/demo/src/modules/customization/dialogs/examples/1/prompt

Comment (waterplea) [reply projects/demo/src/modules/app/app.routes.ts:1]: I think we can ignore this one :) It's just an example, might even be good it is named differently so it's easier for us to search through code.

**Pull request merged**